src/SymbolTable.ts

```diff
@@ -0,0 +1,97 @@
+import type { Identifier } from './lexer/Token';
+import type { BscType } from './types/BscType';
+import { DynamicType } from './types/DynamicType';
+import { UninitializedType } from './types/UninitializedType';
+
+export interface BscSymbol {
```

Comment:
Can you move this interface to the bottom so the primary export (i.e. SymbolTable) is at the top of the file?